### ISSUE TITLE
jQuery update?

### ISSUE THREAD

**Issue by Andy-ABTec** [TV-Rename/tvrename#303]
Hi @JoeBiellik 
Sorry to get in touch this way, didn't know how else to get hold of you...

Did anything more happen about the jQuery update to V3 for Leap Day you were talking about? No problem everything seems to be working as it should, it just seems to have dropped of their roadmap...

**Comment by JoeBiellik** [comment]
I've set it so you can now see my email address on my GH profile.

Leap Day recently accepted my PR updating the jQuery to the latest v1. I had completely forgotten to send in my v3 PR, but that's now pending. Once that hopefully goes through I'll open another PR to remove some extra cruft and nastiness.

Do you want me to fix the issue you opened over there? It's a simple offset issue in their JS.

**Comment by Andy-ABTec** [comment]
If you wouldn't mid that would be brilliant! - Bad template make the rest of it look bad...
Ta!

**Comment by JoeBiellik** [comment]
I'll have to wait on them accepting the jQuery upgrade, don't want to have to change it slightly after.

**Comment by Andy-ABTec** [comment]
No problem I'm well out of my depth with some of this stuff. Anytime is good!

BTW do I need to change anything in the tvrename gh pages branch or is it server-side?

**Comment by JoeBiellik** [comment]
No, the layout file will need to updated with the upstream changes, I can make a pull once they are done.

**Comment by Andy-ABTec** [comment]
Done with the email address if you want to hide it again Ta!

**Comment by Andy-ABTec** [comment]
Hi @JoeBiellik, Just to let you know I've got my local copy of Leap day running on jQuery 3 by cribbing your update to default.html and using a local override for  main .js with the "on load" fix.
Thanks for the help!

**Comment by JoeBiellik** [comment]
They have pulled in my changes to main.js, so an override shouldn't be necessary now.

**Comment by Andy-ABTec** [comment]
great, thanks for letting me know...

**Comment by Andy-ABTec** [comment]
This all appears done-dusted - thanks again @JoeBiellik
